assets/themes/restart/js/render_jobad_list_filter.js

```diff
@@ -383,32 +383,10 @@ function tableLoad(isFirstLoad) {
  * @return {string} table detail as HTML
  */
 function detailFormatter(index, jobAd) {
-    var html = [];
-    html.push('<div class="detail-view"><td colspan="3"><span class="detailFooter"></span>');
-    html.push('<div class="detailContent"><div class="row">');
-    html.push('<div class="highlight">');
-
-        html.push('<div class="col-xs-12 col-sm-12 col-md-12">');
-              html.push(jobAd.content);
-        html.push('</div>');
-
-        html.push('<div class="col-xs-12 col-sm-12 col-md-12">');
-        html.push('&nbsp;</div>');
-
-        html.push('<div class="col-xs-12 col-sm-12 col-md-12">');
-            html.push('<div style="padding-left: 0;" class="col-xs-6 col-md-6">');
-            html.push('<a href="' + jobAd.url + '">View original job desription <i class="fa fa-external-link" aria-hidden="true"></i></a>');
-            html.push('</div>');
-            html.push('<div style="padding-right: 0;" class="col-sm-6 col-md-6 text-right">');
-            html.push('<a href="#' + jobAd.id + '">Get shareable link <i class="fa fa-link" aria-hidden="true"></i></a>');
-            html.push('</div>');
-            html.push('</div>'); // div class="col-xs-12 col-sm-12 col-md-12"
-        html.push('<span class="detailFooter">&nbsp;</span>');
-
-    html.push('</div>'); //highlight
-    html.push('</div></div>'); //detailContent row
-    html.push('</td></div>'); //detail-view
-    return html.join('');
+    var source   = document.getElementById("detailFormatter").innerHTML,
+        template = Handlebars.compile(source),
```

Comment:
I think the lines 386 and 387 should be executed only once, when the page loaded/initialized.
Is it correct?
If the answer is yes than pls move these 2 lines info function `$(document).ready(function () { `

Reply:
ok